lib/debt_management_center/debts_service.rb

```diff
@@ -57,6 +57,7 @@ def veteran_has_dependent_debts?
     def debts_with_sorted_histories
       @debts.select do |debt|
         debt['debtHistory'] = sort_by_date(debt['debtHistory'])
+        debt['compositeDebtId'] = "#{debt['deductionCode']}#{debt['originalAR'].to_i}"
```

Comment:
**Refactor (non-blocking):** I think we should make the composite ID string a separate method so that it's easier to read and refactor down the line.

```ruby
#...
    debt['compositeDebtId'] = getCompositeId(debt)
#...

def getCompositeId(debt)
     "#{debt['deductionCode']}#{debt['originalAR'].to_i}"
end
```